ddtrace/tracer/tracer.go

```diff
@@ -191,7 +212,11 @@ func (t *tracer) worker() {
 					break loop
 				}
 			}
+			t.config.statsd.Incr("datadog.trace.flush.count", []string{"reason:shutdown"}, 1)
 			t.flushPayload()
+			t.config.statsd.Incr("datadog.tracer.stopped", nil, 1)
+			t.config.statsd.Close()
+			t.config.statsd = &noopStatsdClient{}
```

Comment:
If we move `t.sendSpanStats()`, we can also move this stuff into that goroutine.

Reply:
It could. Would the goroutine be doing those bits after observing the stopped channel is closed?